spec/sample-data/valid/control-statement-03.json

```diff
@@ -9,7 +9,7 @@
   "control_level": "unknown",
   "sample_date": "2016-01-01",
   "source_url": "http://download.companieshouse.gov.uk/en_pscdata.html",
-  "retrieved_at": "2015-08-11T23:00:08+01:00",
+  "retrieved_at": "2015-08-11 23:00:08 +0100",
```

Comment:
@CountCulture, is that you? Why did that email comment come through as `openc-jenkins-bot`? What email address did you send it from?